packages/realm/src/app-services/App.ts

```diff
@@ -100,8 +130,8 @@ export class App<FunctionsFactoryType = DefaultFunctionsFactory, CustomDataType
   public static userAgent = `RealmJS/${App.deviceInfo.sdkVersion} (${App.deviceInfo.platform}, v${App.deviceInfo.platformVersion})`;
 
   /** @internal */
-  public static get(userInternal: binding.SyncUser) {
-    const app = appByUserId.get(userInternal.identity);
+  public static getAppForUser(userInternal: binding.SyncUser): App {
```

Comment:
Can we rename this to use `By` instead of `For` for consistency?
```suggestion
  public static getAppByUser(userInternal: binding.SyncUser): App {
```